app/locales/en.json

```diff
@@ -161,7 +161,7 @@
     "positive_label": "Positive Cases",
     "positive_symptoms_description": "As a positive COVID you have the option of laboratory results and daily advice",
     "positive_symptoms_label": "Support the positive",
-    "positive_covid_message": "You have turned out to be COVID positive in the laboratory results",
+    "positive_covid_message": "have turned out to be COVID positive in the laboratory results",
```

Comment:
It is better to use the initial in capital letters and the same with spanish

Reply:
I leave in lowercase the first word because before it will go the user nickname, so it'll be like `pedro have turned out to be covid positive.........etc`